queries.json

```
@@ -11,6 +11,11 @@
       "value": "_path==\"dns\" | count() by query | sort -r",
       "description": "Shows all unique DNS queries in the data set with count"
     },
+    {
+      "name": "Top Queried Domains",
```

Nit: I think "top domains" is a snappier name.
```suggestion
      "name": "Top Domains",
```

---

@nwt: Works for me!